lib/prometheus_exporter/instrumentation/periodic_stats.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module PrometheusExporter::Instrumentation
+  class PeriodicStats
+
+    def self.start(*args, **kwargs)
+      frequency = kwargs[:frequency]
+      client = kwargs[:client] || PrometheusExporter::Client.default
```

Comment:
I think this would also work (not that it's a huge improvement 😉)

```suggestion
    def self.start(*args, frequency:, client: nil, **kwargs)
      client ||= PrometheusExporter::Client.default
```

Reply:
I think this works fine, we also would get to get rid of 1 assertion I think.